### DISCUSSION TITLE
Styling: Formatting Report Width to Wide throws attribute error

### DISCUSSION THREAD

**Discussion by harwantgill** [datapane/datapane#157] [Help] [answered]
Hi, 
Thank you for this awesome visualization library. 
Issue: I am unable to set the formatting width attribute to WIDE. 
It's throwing an AttributeError: WIDE. 

Setting it to NARROW or MEDIUM works with no issues. 

Datapane version: 0.12.1

![Report_width_error](https://user-images.githubusercontent.com/50079345/135563685-1db01614-7be2-4d63-88e2-7ff19dca82ff.JPG)

**Comment by mands** [comment, marked as answer]
Hi,

Apologies for the issue, can you try `FULL` instead?

We'll make sure to update the docs and add `WIDE` as an alias for `FULL` also.

**Reply by harwantgill** [reply]
Hi Mands, 

Thank you for the quick turnaround. 
Full works!
